index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    
+
+    <header>
+      <div class="network"> 
+<ul class = "time">
+    <li><img src="./img/↳ Time.svg">
+    </li>
+</ul>
+<ul>
+<li class = "network-list"><img src="./img/Cellular Connection.svg"></li>
+<li class = "network-list"><img src="./img/Wifi.png" ></li>
+<li class = "network-list"><img src="./img/Battery.svg"></li>
+</ul>     
+</div>    
+        <nav class="nav-links">
+            <p><img src="./img/logo.svg.jpg" height = "50px"></p>
+            <ul class = "nav-list">
+                <li class = "list-item"> <a href="#">Portfolio</a></li>
+                <li class = "list-item"><a href="#">About</a></li>
+                <li class = "list-item"><a href="#">Contact</a></li>
+            </ul>
```

Comment:
- [ ] I would recommend you remove this section and make your UI look close to the Figma design.

### Your Design
![image](https://user-images.githubusercontent.com/86200356/212292390-7e0f76c5-63e3-42f9-a8d7-ca41b83fbbcf.png)

### Figma design
![image](https://user-images.githubusercontent.com/86200356/212292918-69774276-af73-4277-a2a8-9a6ca33a282c.png)